azure-gaming-pr/reference-architectures/cognitive-text-to-speech.md

```diff
@@ -11,7 +11,7 @@ ms.prod: azure-gaming
 
 # Text to Speech Reference Architecture
 
-Help bring everyone into the conversation by converting text messages to audio using **text to speech**.
+Help bring everyone into the conversation by converting text messages to audio using **text to speech** for scenarios, such as game dialogue prototyping, greater accessibility, or NPC voices. Text-to-Speech includes prebuilt neural voice and custom neural voice features. Prebuilt neural voice can provide highly natural out-of-box voices with leading voice variety in terms of a large portfolio of languages and voices. Custom neural voice is an easy-to-use self-service for creating a highly natural custom voice. For details on the two features and voice samples, see [Text-to-Speech overview](https://docs.microsoft.com/azure/cognitive-services/speech-service/text-to-speech).
 
 This article will describe the architecture used in [this sample on GitHub](https://github.com/Azure-Samples/gaming-cognitive-services-text-to-speech). Keep in mind that the code from this reference architecture is only an example for guidance and there may be places to optimize the code before using in a production environment.
```

Comment:
If you're pointing customers to this sample, it's best to confirm that the code is polished. I recommend getting a full review and update.

Reply:
Are there any links to quickstarts that you can add to this article?

Reply:
> Are there any links to quickstarts that you can add to this article?

Yes. I've added the link to Text-to-Speech quickstart.

Reply:
I noticed some deprecated voices in this code sample (TextToSpeech.cs). I'll sync with speech scientist to review the code sample and update it. Thanks.

Reply:
> If you're pointing customers to this sample, it's best to confirm that the code is polished. I recommend getting a full review and update. Customers will use them. @sally-baolian

According to Melissa,  we recommend redirecting the "sample in Github" directly to Speech repository and we have tested the samples. I've updated the wording and link in the article. Thanks.